dashboard/src/public/index.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+use dashboard\controller\GoogleController;
+use dashboard\model\WebPage;
+use Google\ApiCore\ValidationException;
+
+
+// This tells Php which function to run when classes are asked about
+spl_autoload_register('autoloadClasses');
+
+
+// This loads all the dependencies Specified in composer.json
+include "../../vendor/autoload.php";
+
+/**
+ * This loads all the classes used in the project,
+ * @param $class
+ */
+function autoloadClasses($class)
```

Comment:
Should we specify visibility here?

```suggestion
public function autoloadClasses($class)
```

Reply:
Nej, det er ikke i en klasse, så der må man ikke bruge access modifiers